plugins/swagger-apibody/README.md

```diff
@@ -0,0 +1,27 @@
+# @amplication/plugin-<name>
+
+[![NPM Downloads](https://img.shields.io/npm/dt/@amplication/plugin-auth-basic)](https://www.npmjs.com/package/@amplication/plugin-auth-basic)
+
+Add Description of Routes and Entity properties to the REST-API Documentation
+
+## Purpose
+
+the plugin add routes body to each controller
+
+## Configuration
+
+no configuration required
+
+## Scripts
+
+### `build`
+
+Running `npm run build` will bundle your plugin with Webpack for production.
+
+### `dev`
+
+Running `npm run dev` will watch your plugin's source code and automatically bundle it with every change.
+
```

Comment:
This is not needed - you are describing the scripts of the plugins, the idea was to describe scripts that the user may need to use